.ko.yaml

```diff
@@ -0,0 +1,2 @@
+# Use :nonroot base image for all containers
+defaultBaseImage: gcr.io/distroless/base:debug-nonroot
```

Comment:
IIRC, this will also be used for kn image that is built nightly and for released versions and we wanted to use a better base image following the discussion from https://github.com/knative/client/issues/949

I think we can use https://github.com/google/ko#overriding-the-base-for-particular-imports here to override base image for kn like
```
defaultBaseImage: gcr.io/distroless/base:debug-nonroot
baseImageOverrides:
  knative.dev/client/cmd/kn: docker.io/library/alpine:latest
```